auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/Crn.java

```diff
@@ -10,6 +12,11 @@
  */
 public class Crn {
 
+    private static final Pattern CRN_PATTERN =
```

Comment:
I think this pattern isn't quite right. We use:

  private static final Pattern CRN_PATTERN =
      Pattern.compile("^crn:(\\w+):(\\w+):(\\S+):(\\S+):(\\w+):(\\S+)$")

Check out Crn.java in the thunderhead repo.

Reply:
Maybe just copy the Crn class?

Reply:
Yes good idea, this regexp is more elegant than what we have.